{{cookiecutter.project_name|replace(" ", "")}}/Taskfile.yml

```diff
@@ -243,29 +244,16 @@ tasks:
 
   publish:
     desc: Publish the project artifacts; docker images, compiled binaries, etc.
-    vars:
-      INPUT_FILE:
-        # Find the latest tar file for cross-platform builds
-        sh: '{{ '{{if and (ne .LOCAL_PLATFORM .PLATFORM) (ne .PLATFORM "all")}}' }}ls {{ '{{.IMAGE_NAME | replace "/" "_"}}' }}_*_{{ '{{.PLATFORM | replace "/" "_" | replace "," "_"}}' }}.tar 2>/dev/null | sort -r | head -1{{ '{{end}}' }}'
-    preconditions:
-      - sh: which docker
-        msg: "docker is required for publishing"
+    requires:
+      vars:
+        - VERSION
     cmds:
-      - |
-        {{ '{{if or (eq .LOCAL_PLATFORM .PLATFORM) (eq .PLATFORM "all")}}' }}
-          docker push {{ '{{.IMAGE_NAME}}:{{.VERSION}}' }}
-          docker push {{ '{{.IMAGE_NAME}}:latest' }}
-        {{ '{{else}}' }}
-          echo "Publishing cross-platform image from {{ '{{.INPUT_FILE}}' }}"
-          docker run --rm \
-            -v "$(pwd):/src" \
-            -w /src \
-            quay.io/skopeo/stable:latest \
-            copy oci-archive:{{ '{{.INPUT_FILE}}' }} docker://{{ '{{.IMAGE_NAME}}:{{.VERSION}}' }}
-          docker run --rm \
-            -v "$(pwd):/src" \
-            -w /src \
-            quay.io/skopeo/stable:latest \
-            copy oci-archive:{{ '{{.INPUT_FILE}}' }} docker://{{ '{{.IMAGE_NAME}}:latest' }}
-        {{ '{{end}}' }}
```

Comment:
I actually really like this approach but the problem is with creds/auth, as well as this would preclude us from using docker's provenance builder in the future. Maybe reconsider later